composer.json

```diff
@@ -13,27 +13,25 @@
     }
   ],
   "require": {
-    "drupal/auto_entitylabel": "3.x-dev",
-    "drupal/entity_print": "2.x-dev",
-    "drupal/views_block_filter_block": "^1.0@beta",
+    "drupal/auto_entitylabel": "3.x-dev#1dd28579f4559cc03d09ab20143b1feb48644fac",
+    "drupal/ds": "^3.5",
+    "drupal/entity_print": "^2.1",
```

Comment:
@boznik this changed from a dev version. 2.1 seems to be working but do you need the latest dev for something?

Reply:
this is fine